bgpd/bgpd.h

```diff
@@ -1448,6 +1480,10 @@ struct peer {
 	/* Sender side AS path loop detection. */
 	bool as_path_loop_detection;
 
+	/* Conditional advertisement */
+	bool advmap_config_change[AFI_MAX][SAFI_MAX];
```

Comment:
why can't we put those flags under peer->af_flags?

Reply:
peer->af_flags[afi][safi] flags are updated during peer_xxxx_set/peer_xxxx_unset on peer[afi][safi].
But peer->advmap_config_change[afi][safi] is used to notify scanner function when there is a change in related ACLs/prefix-lists/route-maps/match-rules.
I have removed the other flags introduced. please find the reason in below comments.